src/v2/view-builder/views/ov/ChallengeAuthenticatorOktaVerifyView.js

```diff
@@ -0,0 +1,22 @@
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import { 
+  Body as ChallengeAuthenticatorDataOktaVerifyPushOnlyView
+} from './ChallengeAuthenticatorDataOktaVerifyPushOnlyView';
+import { Body as ChallengeAuthenticatorDataOktaVerifyView } from './ChallengeAuthenticatorDataOktaVerifyView';
+
+export default BaseAuthenticatorView.extend({
+  initialize() {
+    BaseAuthenticatorView.prototype.initialize.apply(this, arguments);
+    if (this.isPushOnlyFlow()) {
+      this.Body = ChallengeAuthenticatorDataOktaVerifyPushOnlyView;
+    } else {
+      this.Body = ChallengeAuthenticatorDataOktaVerifyView;
+    }
+  },
+
+  isPushOnlyFlow() {
+    const methodType = this.options.appState.getSchemaByName('authenticator.methodType');
+    const methodTypeOptions = methodType?.options;
```

Comment:
can get rid of methodTypeOptions as well..

Reply:
Didnt get it. could you explain?